data/types/base/bolus/extended/extended.go

```diff
@@ -28,15 +28,28 @@ func SubType() string {
 	return "square"
 }
 
-func New() (*Extended, error) {
-	extendedBolus, err := bolus.New(SubType())
-	if err != nil {
-		return nil, err
-	}
+func NewDatum() data.Datum {
+	return New()
+}
+
+func New() *Extended {
+	return &Extended{}
+}
+
+func Init() *Extended {
+	extended := New()
+	extended.Init()
+	return extended
+}
+
+func (e *Extended) Init() {
```

Comment:
We use the `NewDatum` global func and the `Init` struct func in the data factory. (It uses `NewDatum` first depending upon the type/subType and then invokes `Init` on the newly created object.) We use the `New` and `Init` global funcs in the tests.  So, yes, we use all of these outside of this package.